demos/abstract_storage/ingest_data.py

```diff
@@ -0,0 +1,72 @@
+# Databricks notebook source
+dbutils.widgets.removeAll()
+
+# COMMAND ----------
+
+from dtflw import init_inputs, init_outputs
+
+inputs = init_inputs("Orders", "OrderDetails")
+outputs = init_outputs("Orders", "OrderDetails")
+
+# COMMAND ----------
+
+import dataflow
+
+storage = dataflow.get_storage()
+
+# COMMAND ----------
+
+orders_raw_df = (
+  spark.read
```

Comment:
is it worth adding storage.read_file method?

Reply:
There is `FileStorageBase.read_table` already. 
But in ingest notebooks for reading raw data it is not used, since raw data can be from any source and of any format. `read_table` is used in  in downstream notebooks.